### ISSUE TITLE
Clarifying Style/MultilineOperationIndentation

### ISSUE THREAD

**Issue by faucct** [rubocop-hq/rubocop#3832]
Hello, `Style/MultilineOperationIndentation` is forcing me to align the expression this way though I configure its style as `indented`. Can, please, anyone clarify if this is the desired behaviour and why?

--------

## Expected behavior

```ruby
reminder =
  @current_user.reminders.awaiting_notification.find_by(program_event: @event) ||
    respond_not_found!
```

## Actual behavior

```ruby
reminder =
  @current_user.reminders.awaiting_notification.find_by(program_event: @event) ||
  respond_not_found!
```

**Comment by jonas054** [comment]
The ruby-mode in Emacs indents the same way as RuboCop does. That's reason enough for me to keep it as it is.

**Comment by faucct** [comment]
Well, RubyMine indents it the way I expect it to and is also widely used. I think that this needs a configuration option. I could do it, do you mind?

**Comment by jonas054** [comment]
No, go ahead!

**Comment by wyde19** [comment]
Does that seem like fixed? Should we close that? @jonas054 
https://github.com/bbatsov/rubocop/blob/04087f9e94242468e0da89d211160c8c770f977f/config/default.yml#L461

**Comment by Drenmi** [comment]
@wyde19: I tried it. Still registers an offense on the example code. It seems this configuration option has been there from the start, but it does not cover the case mentioned.

**Comment by AlexWayfer** [comment]
I also affected by this bug. :sweat:

**Comment by bbatsov** [comment]
This cop certainly needs better documentation, that's for sure.
